router/http.go

```diff
@@ -84,23 +89,57 @@ func (s *HTTPListener) Start() error {
 		s.cookieKey = &[32]byte{}
 	}
 
-	started := make(chan error)
-
 	// TODO(benburkert): the sync API cannot handle routes deleted while the
```

Comment:
it's still a problem until routing state (`domains` & `routes` maps) are entirely rebuilt on each call to `Sync`.